flexmeasures/api/common/schemas/pagination.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from flask import abort
+from marshmallow import fields
+
+
+class PageField(fields.Integer):
+    """
+    Field that represents a page number or items per page. It de-serializes from the page/per_page number to an integer.
+    """
+
+    def _deserialize(self, page: int, attr, obj, **kwargs) -> int:
+        page = int(page)
+
+        if page < 1:
```

Comment:
This should be in a `validate` function (see other schemas) and raise a `FMValidationError` I believe.

Reply:
I’m not quite clear on what you mean, this isn't exactly a `schema` Could you point me to the specific function and implementation you mentioned? I checked similar Custom Field classes but didn’t find a validate function.

Reply:
You might be right that the `Field` class does not have a `validate` method. Then my comment is only about using the better exception class, like [here](https://github.com/FlexMeasures/flexmeasures/blob/main/flexmeasures/data/schemas/account.py#L69).